capty.py

```
@@ -191,59 +572,375 @@ def __init__(self):
+            if screen:
+                screen_width = screen.get_width()
+                screen_height = screen.get_height()
+                window_width = 500
+                window_height = 50
+                x = (screen_width - window_width) // 2
+                y = screen_height - window_height - 50  # 50px from bottom
+                self.window.move(x, y)
+        except Exception as e:
+            print(f"Error centering window: {e}")
+
+    def on_window_button_press(self, widget, event):
```

Ruff (0.12.2)

Missing return type annotation for public function `on_window_button_press`

Add return type annotation: `bool`

(ANN201)